src/spikeinterface/core/template.py

```diff
@@ -0,0 +1,60 @@
+import json
+from dataclasses import dataclass, field
+
+import numpy as np
+
+from spikeinterface.core.sparsity import ChannelSparsity
+
+
+@dataclass
+class Templates:
+    templates_array: np.ndarray
+    sparsity: ChannelSparsity = None
```

Comment:
```suggestion
    sparsity_mask: np.ndarray(bool) = None
```
We can always instantiate a `ChannelSparsity` if unit_ids and channel_ids are given.
```
get_channel_sparsity(self):
    assert self.unit_ids is not None and self.channel_ids is not None
    return ChannelSparsity(self.sparsity_mask, self.unit_ids, self.channel_ids)
```

Reply:
this could also be constructed at `___post_init__` and stored as `self.sparsity`

Reply:
Done.